### PR TITLE
Init() is optional

### PR DESCRIPTION
**Pull request by EugenWay**
Fix documentation. Now **init()** is optional if there is no init payload

https://github.com/gear-tech/wiki/pull/108

### CODE REVIEW

File: docs/getting-started-in-5-minutes.md [hunk 1]
```diff
@@ -10,148 +10,136 @@ This guide provides a general overview of running smart contracts on the Gear ne
...
 1. For your convenience, it is recommended that you create a dedicated directory for everything Gear-related. The rest of the article will assume that you are using the paths suggested. To create a folder in your home directory and navigate to it, type:
 
-    ```bash
-    mkdir -p gear
-    cd ~/gear
-    ```
+   ```bash
+   mkdir -p gear
+   cd ~/gear
+   ```
```
Comment (shamilsan) [inline docs/getting-started-in-5-minutes.md:20]: Why 3 spaces?
```suggestion
    ```bash
    mkdir -p gear
    cd ~/gear
    ```
```

Comment (EugenWay) [reply docs/getting-started-in-5-minutes.md]: Not my article, but I have removed all unnecessary spaces in my opinion

File: docs/developing-contracts/gear-program.md [hunk 2]
```diff
@@ -24,6 +24,6 @@ pub unsafe extern "C" fn init() {}
 
 ```
 
-The `init()` function is called only once during program initialization. And handles the incoming `init payload` if there is one.
+The `init()` function is called only once during program initialization. And handles the incoming `init payload` if there is one. Optional if there is no `init payload`.
```
[the remark this answers is not in the document]
Comment (EugenWay) [reply docs/developing-contracts/gear-program.md]: fixed

File: docs/getting-started-in-5-minutes.md [hunk 3]
```diff
@@ -10,148 +10,136 @@ This guide provides a general overview of running smart contracts on the Gear ne
...
-    [lib]
-    crate-type = ["cdylib"]
+   [lib]
+   crate-type = ["cdylib"]
 
-    [dependencies]
-    gcore = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
-    gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
+   [dependencies]
+   gcore = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
```
[the remark this answers is not in the document]
Comment (EugenWay) [reply docs/getting-started-in-5-minutes.md]: fixed

File: docs/developing-contracts/gear-program.md [hunk 4]
```diff
@@ -24,6 +24,6 @@ pub unsafe extern "C" fn init() {}
 
 ```
 
-The `init()` function is called only once during program initialization. And handles the incoming `init payload` if there is one.
+Optional `init()` function is called only once during program initialization. And handles the incoming `init payload` if there is one.
 
 The `handle()` function will be called every time the program receives an incoming message. In this case, the `payload` will be processed.
```
Comment (breathx) [inline docs/developing-contracts/gear-program.md:29]: No mentions of optionality here

**Review by breathx — APPROVED** [review]
Otherwise ok 👌

File: docs/developing-contracts/gear-program.md (hunk 4 above)
Comment (EugenWay) [reply docs/developing-contracts/gear-program.md:29]: Optional `init()` means optional  anyway 👍

Comment (EugenWay) [reply docs/developing-contracts/gear-program.md:29]: fixed

**Pull request merged**